test/test_browser.py

```diff
@@ -1881,6 +1881,22 @@ def setup():
     shutil.copy(Path('sub/test.data'), '.')
     self.btest_exit('test_emscripten_async_load_script.c', args=['-sFORCE_FILESYSTEM'])
 
+  @also_with_wasmfs
+  def test_emscripten_overlapped_package(self):
```

Comment:
Perhaps we can instead test the JS API directly?   i.e. write a test that just called `createPath` twice on the same path?   I think there are already some JS API tests for the FS API.

Reply:
Yeah, that makes sense.  I thought instead of a test that just does createPath twice, I would make a test that showed why the difference in behavior was a bug.

Reply:
I think this test is a little too high level for this fix.  I don't think we need/want to involve the file packager to get this testing.   Would it be OK to just update `test_fs_js_api.c` instead of adding this new test?

Reply:
Even if this test is too high level for this fix, I do think it's an important regression test to have since having multiple packages is valuable and there's no obvious reason they couldn't e.g. mount to /assets/pack-a and /assets/pack-b.  Can we keep it there as well as the js_fs_api test?

Reply:
Fair enough.   Lets group it with the other file packager tests then.   How about calling it `test_file_packager_overlapping`?

And can you move it to test_other with all the other file_packager tests (below `test_file_packager_modularize`)

Reply:
Done!